ui/app/src/modules/Content/Dependencies/DependencySelection.tsx

```diff
@@ -294,94 +305,57 @@ export function DependencySelectionDelete(props: DependencySelectionProps) {
           checked={checked}
           setChecked={setChecked}
         />
-        {
-          resultItems == null ? (null) : (
-            <>
-              <SelectionList
-                title={
-                  <FormattedMessage
-                    id="deleteDialog.childItemsText"
-                    defaultMessage={`Child Items`}
-                  />
-                }
-                subtitle={
-                  <FormattedMessage
-                    id="deleteDialog.willGetDeleted"
-                    defaultMessage={` Will get deleted`}
-                  />
-                }
-                uris={resultItems.items1}
-                displayItemTitle={false}
+        <>
+          <SelectionList
+            title={
+              <FormattedMessage
+                id="deleteDialog.childItemsText"
+                defaultMessage={`Child Items`}
               />
-              <SelectionList
-                title={
-                  <FormattedMessage
-                    id="deleteDialog.dependentItems"
-                    defaultMessage={`Dependent Items`}
-                  />
-                }
-                subtitle={
+            }
+            subtitle={
+              <FormattedMessage
+                id="deleteDialog.willGetDeleted"
+                defaultMessage={` Will get deleted`}
+              />
+            }
+            uris={resultItems.childItems}
+            displayItemTitle={false}
+          />
+          <SelectionList
+            title={
+              <FormattedMessage
+                id="deleteDialog.dependentItems"
+                defaultMessage={`Dependent Items`}
+              />
+            }
+            subtitle={
+              <FormattedMessage
+                id="deleteDialog.brokenItems"
+                defaultMessage={` Will have broken references`}
```

Comment:
Is the space at the beginning intentional? There be a better way of achieving it even if it is.
Please change all these curly brackets with side ticks to regular double quotes